araproc/analysis/standard_reco.py

```diff
@@ -417,7 +431,7 @@ def lookup_arrival_time(self,
                                                                                        )
         return arrival_time
 
-    def __get_correlation_function(self, ch1, ch2, wavepacket):
+    def __get_correlation_function(self, ch1, ch2, wavepacket, applyHilbert):
```

Comment:
I have another finicky thought, I'm going to want to use the data from this functions so the smartest way would be to call `__get_correlation_function` then access the items through `__corr_functions_*` right? That way we don't recalculate the cross correlation over and over? 

Super finicky and over the top, sorry feel free to ignore, but would it make sense to call this `__calculate_correlation_function` that only calculates the cross correlations and saves them to `__corr_functions_*` then having a getter function that only calls `__calculate_correlation_function` if those `__corr_functions_*` are not initialized yet (or equal to `None`) and  returns the function the user requested?

Reply:
That function actually already exists! It's true this isn't actually _just_ a getter function, but it will only calculate the correlation functions if another event's correlation functions are currently cached. Otherwise it will return the cached correlation function that you request. So you never need to access the `__corr_functions_*` directly. Correct usage would be something like `my_corr = __get_correlation_function(0, 1, wavepacket, False)` Another example is in `get_corr_snr`

Reply:
Oh I see, I think I misunderstood it because I was only looking at the diff. Sorry about that. Sounds good to me, thanks!